docs/sources/reference/components/discovery/discovery.relabel.md

```diff
@@ -78,7 +78,9 @@ In those cases, exported fields retain their last healthy values.
 
 ## Debug information
 
-`discovery.relabel` does not expose any component-specific debug information.
+`discovery.relabel` exposes some target-level debug information per target:
+
+* The current label and the relabelled value associated with each target.
```

Comment:
rather than adding doc here, could you instead update the list of supported components in the note of this document please: https://github.com/grafana/alloy/blob/main/docs/sources/troubleshoot/debug.md